packages/backend/src/server/api/endpoints/admin/update-meta.ts

```diff
@@ -296,6 +296,10 @@ export const meta = {
 		objectStorageS3ForcePathStyle: {
 			validator: $.optional.bool,
 		},
+
+		additionalFieldLimit: {
```

Comment:
Should now probably be `additionalFieldLimit: { type: 'integer', nullable: true },`, right?